lib/src/functions/math.dart

```diff
@@ -213,6 +214,115 @@ final _sqrt = BuiltInCallable("sqrt", r"$number", (arguments) {
   return SassNumber(math.sqrt(number.value));
 });
 
+///
+/// Trigonometric functions
+///
+
+final _acos = BuiltInCallable("acos", r"$number", (arguments) {
+  var number = arguments[0].assertNumber("number");
+  number.assertNoUnits();
+  var numberValue = fuzzyEquals(number.value.abs(), 1)
+      ? fuzzyRound(number.value)
+      : number.value;
+  var acos = math.acos(numberValue) * 180 / math.pi;
+  return SassNumber.withUnits(acos, numeratorUnits: ['deg']);
+});
+
+final _asin = BuiltInCallable("asin", r"$number", (arguments) {
+  var number = arguments[0].assertNumber("number");
+  number.assertNoUnits();
+  var numberValue = number.value;
+
+  if (fuzzyEquals(numberValue, 0) && numberValue.isNegative) {
```

Comment:
Why don't we round a positive number that fuzzy-equals 0?

Reply:
Good question. The spec only specifies this behavior for `-0.0`. But it would probably be more consistent if the spec also handled `0`.

Reply:
The same would apply to `sqrt()`, which currently only handles `-0.0`.